spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/UiController.java

```diff
@@ -28,9 +28,11 @@
 @Controller
 public class UiController {
 
-	@RequestMapping("/dashboard")
+	public static final String WEB_UI_INDEX_PAGE_NAME = "dashboard";
```

Comment:
I would make the starting slash part of the constant, that 1) makes more sense (it's a part, not just a "word") and 2) would simplify its usage quite a bit